striprtf/striprtf.py

```diff
@@ -149,21 +153,18 @@ def rtf_to_text(text, encoding="utf-8", errors="strict"):
             curskip = 0
             if char == "~":
                 if not ignorable:
-                    out = out + b"\xA0"  # NBSP
+                    out += b"\xA0".decode('utf-8')
```

Comment:
This seems incorrect.  `decode` assumes that  `\xA0` is UTF-8, which it is *not*.    It is the Unicode codepoint for [U+00A0](https://www.fileformat.info/info/unicode/char/00a0/index.htm) (non-breaking space).

I think you want
```py
out += chr(0xa0).encode('utf-8') # NBSP
```
here, or simply hard-code the UTF-8 encoding:
```py
out += b'\xc2\xa0' # NBSP in UTF-8
```